packages/fuchsia_remote_debug_protocol/lib/fuchsia_remote_debug_protocol.dart

```diff
@@ -0,0 +1,15 @@
+// Copyright 2018 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// Provides an API to test/debug Flutter applications on remote Fuchsia devices
+/// and emulators.
+///
+/// The application typically runs in a separate process from the actual
+/// test, wherein the user can supply the process with various events to test
```

Comment:
if it's on a remote device, isn't it guaranteed to be in a separate process?

Reply:
I don't remember the particulars, but I remember being told it won't always be on a remote device.